engine/modelc/src/test/test_model.cpp

```diff
@@ -95,9 +96,14 @@ TEST(ModelGLTF, LoadSkeleton)
 
 static int TestStandalone(const char* path)
 {
+    uint64_t tstart = dmTime::GetTime();
+
     dmModelImporter::Options options;
     dmModelImporter::Scene* scene = LoadScene(path, options);
 
+    uint64_t tend = dmTime::GetTime();
+    printf("Model %s loaded in %.3f seconds.\n", path, float(tend-tstart)/1000000.0f);
+
```

Comment:
Our model test doubles as a first line of testing the importing a single file and outputting the statistics.
Easy to debug and run, and always available.